packages/datadog_flutter_plugin/android/src/main/kotlin/com/datadoghq/flutter/DatadogSdkPlugin.kt

```diff
@@ -102,10 +103,9 @@ class DatadogSdkPlugin : FlutterPlugin, MethodCallHandler {
             config.trackingConsent
         )
 
-        if (config.loggingConfiguration != null) {
-            logsPlugin = DatadogLogsPlugin()
-            logsPlugin?.setup(binding, config.loggingConfiguration!!)
-        }
+        // Always setup logging as a user can create a log after initialization
+        logsPlugin = DatadogLogsPlugin()
+        logsPlugin?.setup(binding)
```

Comment:
```suggestion
        logsPlugin = DatadogLogsPlugin().apply { setup(binding) }
```

Reply:
Should that pattern be carried down into the other setup calls?

Reply:
As you wish :) It is a nitpick to me and doesn't really matter in the single-threaded environment.